### PR TITLE
add tucana

### PR DESCRIPTION
**Pull request by David885L2**
add tucana to explorer.

### CODE REVIEW

File: testnets/tucana/chain.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        "average_gas_price": 0.15,
+        "high_gas_price": 0.4
+      },
+      {
+        "denom": "ueth"
+      },
+      {
+        "denom": "uusdc"
+      }
+    ]
```
Comment (ALPAC-4) [inline testnets/tucana/chain.json:29]: I think your chain doesn't have `ueth` and `uusdc` can you remove this?

File: testnets/tucana/chain.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      {
+        "denom": "utuc"
+      },
+      {
+        "denom": "move/dbf06c48af3984ec6d9ae8a9aa7dbb0bb1e784aa9b8c4a5681af660cf8558d7d"
+      },
+      {
+        "denom": "move/a2b0d3c8e53e379ede31f3a361ff02716d50ec53c6b65b8c48a81d5b06548200"
+      }
+    ]
```
Comment (ALPAC-4) [inline testnets/tucana/chain.json:42]: your chain doesn't have a staking module can you make this to empty array?

File: testnets/tucana/chain.json [hunk 3]
```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "tucana",
+  "chain_id": "tucana-69",
+  "website": "https://tucana.zone",
+  "pretty_name": "Tucana",
+  "status": "live",
+  "network_type": "testnet",
```
Comment (ALPAC-4) [inline testnets/tucana/chain.json:8]: mahalo-2 is a devnet, can you change it to devnet?
and can you move the tucana folder to devent?

File: testnets/tucana/chain.json [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+      {
+        "address": "https://tucana-lcd.testnet.mesoops.net",
+        "provider": "Foundation"
+      }
+    ],
+    "api": [
+      {
+        "address": "https://tucana-api.testnet.mesoops.net",
+        "provider": "Foundation"
+      }
```
Comment (ALPAC-4) [inline testnets/tucana/chain.json:81]: api field is for minitia/initia indexer api and you chain is using the kv store version which means you don't have to fill this. can you remove the api field?

File: testnets/tucana/chain.json [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+      "url": "https://explorer.mahalo-2.initia.xyz/?layer=tucana",
+      "tx_page": "https://explorer.mahalo-2.initia.xyz/transactions?layer=tucana",
+      "account_page": "https://explorer.mahalo-2.initia.xyz/address/${accountAddress}?layer=l1%20testnet"
+    },
+    {
+      "kind": "initia scan",
+      "url": "https://scan.initia.xyz/mahalo-2",
+      "tx_page": "https://scan.initia.xyz/mahalo-2/txs/${txHash}",
+      "account_page": "https://scan.initia.xyz/mahalo-2/accounts/${accountAddress}"
+    }
```
Comment (ALPAC-4) [inline testnets/tucana/chain.json:102]: can you change these endpoints to `https://scan.initia.xyz/tucana-69` ?

File: testnets/tucana/chain.json [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+      "png": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/devnets/initia/images/INIT.png",
+      "svg": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/devnets/initia/images/INIT.svg"
+    }
+  ],
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/devnets/initia/images/INIT.png",
+    "svg": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/devnets/initia/images/INIT.svg"
+  },
+  "metadata": {
+    "is_l1": true,
```
Comment (ALPAC-4) [inline testnets/tucana/chain.json:121]: can you chage this to false or remove it?

File: testnets/tucana/chain.json [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+        "channel_id": "channel-1",
+        "version": "ics20-1"
+      },
+      {
+        "chain_id": "miniwasm-2",
+        "port_id": "nft-transfer",
+        "channel_id": "channel-3",
+        "version": "ics721-1"
+      }
+    ],
```
Comment (ALPAC-4) [inline testnets/tucana/chain.json:148]: can you make this empty?

File: testnets/tucana/chain.json [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+    "assetlist": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/devnets/initia/assetlist.json"
```
Comment (ALPAC-4) [inline testnets/tucana/chain.json:149]: can you change this to `https://raw.githubusercontent.com/initia-labs/initia-registry/main/devnets/tucana/assetlist.json`

**Review by ALPAC-4 — COMMENTED** [review]
And can you remove assetlist.json? Please make file after content include. ibc data too.

**Review by ALPAC-4 — COMMENTED** [review]
https://github.com/David885L2/tucana/pull/1 I made pr to your repo can you merge it?

**Comment by David885L2** [comment]
close

**Pull request closed without merging**